src/main/ml-config/security/users/esri-connector-unauthorized-user.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Since this is purely for testing, I think this should be under src/test/ml-config/security instead of under main. It would be a bit weird to have this user created for the normal install of the connector.